xpad.c

```diff
@@ -1379,7 +1379,7 @@ static int xpad_start_xbox_360(struct usb_xpad *xpad)
 	status = usb_control_msg(xpad->udev,
 		usb_rcvctrlpipe(xpad->udev, 0),
 		0x1, 0xc1,
-		cpu_to_le16(0x100), cpu_to_le16(0x0), data, cpu_to_le16(20),
+		cpu_to_le16(0x100), cpu_to_le16(0x0), data, cpu_to_le16(14),
```

Comment:
This value should be 20 or 0x14 (as that's the same numerically). This isn't a correct "fix".

Reply:
I don't know, but I know that with that change now my game controller starts without using https://gist.github.com/dnmodder/de2df973323b7c6acf45f40dc66e8db3 the same with 0x14 also works.

Reply:
Without the change, I assure you it doesn't work, and I guess it happens to more people because someone wrote fixcontroller.py.
I do not say that I change this change is correct at all, but I have based on fixcontroller.py, since with this script my game controller and that of the people who have posted there works.

Reply:
no, if 0x14 works, 20 must work too. So this change cannot be the fix.

Reply:
Let's see if I can try later with different values there, as I don't have the controller here.